cyder/cydhcp/network/models.py

```diff
@@ -288,6 +287,16 @@ def descendants(self):
                 'Network.descendants does not currently support IPv6')
 
     @property
+    def children(self):
+        self.update_network()
+        descendants = set(self.descendants)
+        children = []
+        for d in descendants:
+            if d.parent not in descendants:
```

Comment:
This implementation seems a bit obtuse. Is there a reason not to use `d.parent is self` or `d.parent == self`?